skbio/stats/composition.py

```diff
@@ -1235,55 +1325,65 @@ def ancom(
 
     See Also
     --------
-    multiplicative_replacement
+    multi_replace
     scipy.stats.ttest_ind
     scipy.stats.f_oneway
     scipy.stats.wilcoxon
     scipy.stats.kruskal
 
+    Warnings
+    --------
+    ``multiple_comparisons_correction`` is deprecated as of ``0.6.0``. It has
+    been renamed to ``p_adjust``.
+
+    ``significance_test=None` is deprecated as of ``0.6.0``. The default value
```

Comment:
Missing second tick mark after "significance_test=None". This causes rendering issues.